matrix_sdk_crypto/src/store/memorystore.rs

```diff
@@ -165,6 +191,11 @@ impl CryptoStore for MemoryStore {
         !self.users_for_key_query.is_empty()
     }
 
+    fn users_for_key_query(&self) -> HashSet<UserId> {
+        #[allow(clippy::map_clone)]
```

Comment:
Is there a bug report about this generating false positives?

Reply:
No idea.